packages/core/fluxcd/templates/flux-instance.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: fluxcd.controlplane.io/v1
+kind: FluxInstance
+metadata:
+  name: flux
+spec:
+  cluster:
+    domain: cozy.local
```

Comment:
Is it possible to not hardcode the domain? Users might use the different one

Reply:
We can make our own local values in the parent chart, and keep them for the `FluxInstance` - sure, I think so.

Reply:
Everything that happens in the chart now is either a parameter or an override. 🎸 🚀 🪨

Reply:
Maybe we can figure out how to pass-through to accept the domain as a parameter from the `cozystack` ConfigMap, before this merges - this will help me know how to add the `distribution.registry` later in a future built 👍

Reply:
@stefanprodan provided an elegant solution for removing cluster domain dependency from fluxcd:
https://github.com/controlplaneio-fluxcd/flux-operator/issues/34#issuecomment-2171122183

I'd like to go this way right now as we won't break the exisiting user flow.

But it seems we still have to add `cluster-domain` and `kubernetes-endpoint` options into cozystack configmap.
So if specified, it must be used by fluxcd and other applications, because in some cases this [might be required](https://github.com/controlplaneio-fluxcd/flux-operator/issues/34#issuecomment-2171088661).

I'll prepare an issue to keep this in mind.

Reply:
done https://github.com/aenix-io/cozystack/issues/168